lib/components/table.vue

```diff
@@ -441,7 +438,18 @@
                 }
 
                 // Shallow copy of items, so we don't mutate the original array order/size
-                items = items.slice();
+
+                items = JSON.parse(JSON.stringify(items))
```

Comment:
This would no longer be a shallow copy of items, but a deep copy (which means much more memory usage as a side effect, and slightly slower than a slice)